src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -120,4 +126,31 @@ private function findRouteNameContaining(string $name, RouteCollection $routes):
 
         return $foundRoutesNames;
     }
+
+    private function sortRoutes($routes, $propertyName)
+    {
+        $sortedRoutes = $routes->all();
+        if ('name' === $propertyName) {
+            ksort($sortedRoutes);
+        }
+        if ('path' === $propertyName) {
+            $sortedRoutesByPath = [];
```

Comment:
Couldn't this be achieved with `usort` and `<=>` operator with much less lines of code?

Reply:
Thanks @vudaltsov, personally I find `ksort` fast enough in this scope, but I'll be happy to apply a patch.

Reply:
@OskarStark @vudaltsov @nicolas-grekas While I work on the `usort `patchm to see if I can make the sorting simpler and faster, I would like your opinion about adding a disclaimer before the routes list, to point out that the sorted results don't reflect the parsing order.

Reply:
> Couldn't this be achieved with `usort` and `<=>` operator with much less lines of code?

Fixed via d41ed89